README.md

```diff
@@ -1,27 +1,23 @@
 # PHP Demandware XML
-A PHP library for working with Demandware XML files.
-
-**Exporting:** Supports category, product, variant and assignment files and allows elements to be added in the order that makes sense for your code - they'll be exported in the sequence specified by the XSD automatically, and custom attribute elements are ordered by attribute ids for consistency between exports
 
-**Validation**: Validates against included XSD schemas (done automatically when exporting) but only for files up to 1Gb.
+A PHP library for working with Demandware XML files.
 
-**Parsing:** Simple methods to retrieve category, assignment, product, variation, set and bundle information from an XML file, either as an array, or yield using generators. Maps IDs to the more useful elements and, optionally, attributes.
+**Exporting**: Supports category, product, variant and assignment files and allows elements to be added in the order that makes sense for your code - they'll be exported in the sequence specified by the XSD automatically, and custom attribute elements are ordered by attribute ids for consistency between exports.
 
-**Comparison**: *in-progress* Reports on different IDs, elements and attributes, regardless of how they're ordered in the files (i.e. higher-level than diff), which is useful when different systems/processes have exported the files.
+**Validation**: Automatically validates files up to 1Gb (against the included XSD schemas) after exporting.
 
-## Install
-Via Composer:
+**Parsing**: Retrieve category, assignment, product, variation, set and bundle information from an XML file, either as an array, or yield using generators. Maps IDs to the more useful elements and, optionally, attributes.
 
-``` bash
-$ composer require fusionspim/php-demandware-xml
-```
+## Installation
 
+Run `composer require --dev fusionspim/php-demandware-xml`.
```

Comment:
Why `--dev` ?

Reply:
No idea! Was like that before and didn't spot it. Have removed ` --dev` ta.